src/tests/test-utilities/FormTester.cs

```diff
@@ -26,6 +26,8 @@
 using System.Windows.Forms;
 using NUnit.Framework;
 
+[assembly: NUnit.Framework.NonTestAssembly]
```

Comment:
why would we have this here in a test instead of the assembly information? Also 
```suggestion
[assembly: NonTestAssembly]
```
interesting feature

Reply:
I agree with the suggested change but I'll fix it along with other stuff rather than using the Beta feature, which would seem to create lots of extra commits.

This should really be in an AssemblyInfo file, but this project doesn't have any at the moment. I could add them at least to the two projects that have the attribute.

As commented above, I'm more or less misusing the feature so our tests pass when you run the solution file. It's intended to be used in FormTester, which references the framework but is not a test file. It's not really aimed at mock-assembly, but it works there.